src/main/java/org/skriptlang/skript/lang/structure/Structure.java

```diff
@@ -169,6 +174,16 @@ public Priority getPriority() {
 		return DEFAULT_PRIORITY;
 	}
 
+	@Override
+	public final List<Class<? extends Section>> getUsableSections() {
```

Comment:
Very nitpick (sorry :() but we could probably have this return the default value (e.g. just call to super)

Reply:
this would look a little weird, `super` itself wont work so it would need to be `SyntaxElement.super.getUsableSections()`, would you be okay with this?

Reply:
Sure